tests/openvino/test_modeling.py

```diff
@@ -149,7 +149,7 @@ def __init__(self, *args, **kwargs):
         self.OV_MODEL_ID = "echarlaix/distilbert-base-uncased-finetuned-sst-2-english-openvino"
         self.OV_DECODER_MODEL_ID = "helenai/gpt2-ov"
         self.OV_SEQ2SEQ_MODEL_ID = "echarlaix/t5-small-openvino"
-        self.OV_SD_DIFFUSION_MODEL_ID = "hf-internal-testing/tiny-stable-diffusion-openvino"
+        self.OV_SD_DIFFUSION_MODEL_ID = "katuni4ka/tiny-stable-diffusion-openvino"
```

Comment:
I would however suggest we add a warning that notifies the user if a very old IR is being used and prompt them to re export for optimal perf.

Reply:
it is good idea if there is robust way to estimate what is old IR and what is not, but it is not so now... I think I need to think dipper about that and implement separately

Reply:
please open an issue to track this item, I believe an estimator is the one you just used (2022/2023 -> old).